src/openrct2/network/network.h

```diff
@@ -52,6 +51,14 @@ extern "C" {
 
 #ifndef DISABLE_NETWORK
 
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+#include <openssl/evp.h>
```

Comment:
I can even remove the #ifdef, because openssl/evp.h already contains a __cplusplus guard.